cleanlab/classification.py

```diff
@@ -490,18 +572,18 @@ def find_label_issues(
 
         Returns
         -------
-        label_issues_mask : np.array
-          A boolean mask for the entire dataset where ``True`` represents a
-          label issue and ``False`` represents an example that is accurately
-          labeled with high confidence.
+        label_issues_df : pd.DataFrame
+          DataFrame with same format as the one returned by :py:meth:`CleanLearning.fit()
+        <cleanlab.classification.CleanLearning.fit>`.
+          See there for documentation regarding column definitions.
```

Comment:
I think you sould move the return docstring for label_issues_df  here since THIS is where the df actually gets returned and just refer to it here in fit()?

Reply:
Again, I really think you should provide a spec for the df thats returned since this is the method to create that df and return it and fit doesnt even return it